libpod/adapter/client.go

```diff
@@ -8,7 +8,8 @@ import (
 
 // Connect provides a varlink connection
 func (r RemoteRuntime) Connect() (*varlink.Connection, error) {
-	connection, err := varlink.NewConnection("unix:/run/podman/io.podman")
+	//connection, err := varlink.NewConnection("unix:/run/podman/io.podman")
```

Comment:
might want to clear this line at some point

Reply:
yes but worth tidying up, ty